db/migrate/20170427152006_remove_invalid_hawkular_endpoints.rb

```diff
@@ -0,0 +1,29 @@
+class RemoveInvalidHawkularEndpoints < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Endpoint < ActiveRecord::Base; end
+
+  def up
+    Endpoint.where(
+      :resource_type => "ExtManagementSystem",
+      :role          => 'hawkular',
+      :hostname      => nil
+    ).delete_all
+  end
+
+  def down
+    ExtManagementSystem.where(
+      :type => %w(ManageIQ::Providers::Openshift::ContainerManager ManageIQ::Providers::Kubernetes::ContainerManager)
+    ).each do |ems|
+      Endpoint.create!(
```

Comment:
what about providers that have both endpoints?
wouldn't this create a 3rd one going back?